Resources/public/js/views/dashboard/plugins/ez-base-dashboardblockviewplugin.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-base-dashboardblockviewplugin', function (Y) {
+    'use strict';
+
+    /**
+     * Provides the dashboard base block view plugin class
+     *
+     * @module ez-base-dashboardblockviewplugin
+     */
+    Y.namespace('eZ.Plugin');
+
+    /**
+     * The dashboard base block view plugin
+     *
+     * @namespace eZ.Plugin
+     * @class DashboardBlockBaseView
+     * @constructor
+     * @extends Base
+     */
+    Y.eZ.Plugin.DashboardBlockBaseView = Y.Base.create('baseDashboardBlockViewPlugin', Y.Base, [], {
```

Comment:
that's not a bad idea but that's dead code and this is not needed right now again don't add such code. Also if that was really needed, this should have been tested.